src/app/drive/components/DeleteBackupDialog/DeleteBackupDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import errorService from 'app/core/services/error.service';
+import { uiActions } from 'app/store/slices/ui';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { RootState } from 'app/store';
+import { DriveFolderData as DriveWebFolderData, DriveItemData } from '../../types';
+import { deleteItemsThunk } from '../../../store/slices/storage/storage.thunks/deleteItemsThunk';
+import { deleteBackupDeviceAsFolder } from '../../../drive/services/folder.service';
+import { backupsThunks } from 'app/store/slices/backups';
+import { SdkFactory } from '../../../core/factory/sdk';
+import { BreadcrumbItemData } from 'app/shared/components/Breadcrumbs/Breadcrumbs';
+import Dialog from '../../../shared/components/Dialog/Dialog';
+interface DeleteBackupDialogProps {
```

Comment:
nit: add a blank line between imports and the following code

Reply:
added